opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -315,25 +316,25 @@ def _shutdown():
 
         if not did_shutdown:
             _logger.warning("shutdown can only be called once")
-            return False
+            return
 
-        overall_result = True
+        metric_reader_errors = []
 
         for metric_reader in self._sdk_config.metric_readers:
-            metric_reader_result = metric_reader.shutdown()
+            try:
+                metric_reader.shutdown()
 
-            if not metric_reader_result:
-                _logger.warning(
-                    "MetricReader %s failed to shutdown", metric_reader
-                )
+            # pylint: disable=broad-except
+            except Exception as error:
 
-            overall_result = overall_result and metric_reader_result
+                metric_reader_errors.append(error)
```

Comment:
We no longer log which metric reader had an issue

Reply:
Fixed, added the metric reader to the error message.